src/hotspot/share/prims/whitebox.cpp

```diff
@@ -2684,6 +2692,8 @@ static JNINativeMethod methods[] = {
                                                       (void*)&WB_GetMethodStringOption},
   {CC"getDefaultArchivePath",             CC"()Ljava/lang/String;",
                                                       (void*)&WB_GetDefaultArchivePath},
+  {CC"getCDSGenericHeaderMinVersion",     CC"()I",    (void*)&WB_GetCDSGenericHeaderMinVersion},
+  {CC"getCurrentCDSVersion",              CC"()I",    (void*)&WB_GetCDSCurrentVersion},
```

Comment:
I think should also be inside a `#if INCLUDE_CDS` block.

Also, I think the cds.h should be included explicitly; currently it is included transitively via filemap.hpp.

Pre-existing issue: below CDS specific include statements should be inside a `#if INCLUDE_CDS` block.
```
  27 #include "cds/cdsConstants.hpp"
  28 #include "cds/filemap.hpp"
  29 #include "cds/heapShared.inline.hpp"
  30 #include "cds/metaspaceShared.hpp"
```

Reply:
I will add include "cds.h". Those including, should be cleaned up in a separate issue.